src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -25,6 +25,7 @@ const _ = require('lodash');
     }
 
     function changeParticipation(partstat) {
+      $rootScope.$emit(CAL_EVENTS.UPDATE_ACTION_EXCAL, self.linksMapping[partstat]);
```

Comment:
`calendar` and `excal` are 2 different SPAs, why there is a angular event beween them?

Reply:
excal component use calendar component to display event detail (eventCalView) , 
the event is fired in calendar component (the parent) for that we must emit event on it ,
 after that the child component (excal) keep listening to upcoming event t

Reply:
I mean if the logic is used by only `excal` SPA, then it is not `common` libs anymore.
Furthermore, why external user view code is maintained in `calendar` codebase?
So, I suggest we move this template and her logic to `excal` module. WDY?

Reply:
i think is not the right moment to refactor this , eventCalView is used in inbox calendar (to send invitation)

Reply:
As you wish. However, a ticket should be created to refactor it.

Reply:
ok wil discuss with @renaudboyer  @huy-ta  about it